doc/package-notes.xml

```diff
@@ -464,7 +464,6 @@ it. Place the resulting <filename>package.nix</filename> file into
     or because they cannot be patched.The steps to launch a game 
     directly are: 
     <orderedlist>
```

Comment:
Maybe we want to just drop this whole section on running the game directly? I can't think of a scenario when it would work better than when it is started from Steam itself; also, advertising manual `patchelf`ing (which would be required to run the game like this, but is a very hacky and non-trivial process) seems a bad idea.